hypex/utils/errors.py

```diff
@@ -28,10 +28,14 @@ def __init__(self, role):
         super().__init__(f"No columns found by role {role}")
 
 
-class ComparisonNotSuitableFieldError(Exception):
+class GroupFieldNotSuitableFieldError(Exception):
     def __init__(self, group_field):
         super().__init__(f"Group field {group_field} is not suitable for comparison")
 
+class TargetFieldNotSuitableFieldError(Exception):
```

Comment:
I was trying to, but the text is different not only in the name of the variable but the rest as well. It can be done with complex ifs, but it seems to be overcomplicated.